### PR TITLE
Added configuring ide for rider

### LINKED ISSUE #6495: Add JetBrains Rider to "Configuring an IDE"
**Your Godot version:**

N/A

**Issue description:**

Currently, the "Configuring an IDE" section mentions [JetBrains CLion](https://www.jetbrains.com/clion/).

However, CLion is not the right JetBrains IDE to contribute to Godot. The vast majority of files are skipped by the parser because they are too large, and using android's CMakeLists is a bit of a hack that doesn't help CLion. It also parses test files in loop, probably lost in the project.

[JetBrains Rider](https://www.jetbrains.com/rider/) is a much better IDE for working on Godot.
- Rider analyses the project using the `.sln` solution that we can generate by following the steps for Visual Studio Community.
- It understands C++ perfectly.
- It doesn't fail parsing project files.
- It doesn't loop on test files.
- We can run the project by using the play button instead of manual terminal compilation, and without additional configuration, because it understands the .sln solution.

Here's the official comparison between Rider and CLion: https://www.jetbrains.com/products/compare/?product=clion&product=rider.

Therefore, I suggest to add JetBrains Rider to the list. However, it is still useful to keep CLion in the list.

**URL to the documentation page:**

https://docs.godotengine.org/en/stable/development/cpp/configuring_an_ide/index.html

### PR DESCRIPTION
**Pull request by br1trs**
Fixes #6495 
I created a configuring IDE for Rider documentation.

### CODE REVIEW

**Comment by skyace65** [comment]
The png images in this PR need to be converted to .webp, it's what we're using now in the docs and converting things to as images need updating. If you need any help with that we have a guide [here](https://docs.godotengine.org/en/latest/contributing/documentation/docs_image_guidelines.html).

Ideally we'd also like PRs to be one commit, info on doing that [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#modifying-a-pull-request).

And lastly I don't think you intended to add what's in your .idea folder to this PR.

**Comment by br1trs** [comment]
Thank you! I will check fix that, and yes that folder shouldn' be there, sorry about that.

File: contributing/development/configuring_an_ide/rider.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Use the root path to the project folder, not ``project.godot`` file path.
+
+.. figure:: img/rider_configurations_changed.webp
+   :align: center
+
+- Finally click on apply and ok to save the changes.
+
+- You can now launch the debugger by pressing the **Debug** icon at the top of the screen and it will open the project to debug.
+
+Alternatively you can use **run > attach to process** to attach the debugger to a running Godot instance.
```
Comment (mhilbrunner) [inline contributing/development/configuring_an_ide/rider.rst:66]:
```suggestion
Alternatively you can use **Run > Attach to Process** to attach the debugger to a running Godot instance.
```

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thanks and congrats on your first merged contribution to Godot's docs! 🎉

**Pull request merged**